corehq/apps/data_interfaces/models.py

```diff
@@ -882,8 +882,7 @@ def when_case_matches(self, case, rule):
             if case_id == case.case_id:
                 continue
             result = update_case(case.domain, case_id, case_properties=properties, close=False,
-                xmlns=AUTO_UPDATE_XMLNS)
-
+                xmlns=AUTO_UPDATE_XMLNS)  # TODO: confirm proper format
```

Comment:
Is this related to this PR?

Reply:
+1 I don't think the `properties` dict passed here includes any actions, so we're unlikely to be intertwined here with the model changes. @stephherbers give me the thumbs up and I'll remove this in my next commit.